apps/src/templates/page.html.ejs

```diff
@@ -68,7 +68,7 @@
         <div id="blockUsed" class=<%= data.blockCounterClass %>>
           <%= data.blockUsed %>
         </div>
-        <span>&nbsp;/</span>
+        <span>/</span>
```

Comment:
this extra space must have been there because the margins used to be off by 5px. It is no longer needed.